db/migrate/20230328131406_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
- [ ] Taking into consideration the given ERD, let's bear in mind that `author` is aliasing the `user` model, and hence, our migration query should be similar to:

```ruby
...
  def change
    add_reference :like, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...

```

or.. 

```ruby
  create_table :likes do |t|
    . . .
    t.references :author, foreign_key: { to_table: 'users' } 
    . . .
  end
```